### PR TITLE
Fix sliders theming for calendar backup settings

### PR DESCRIPTION
**Pull request by sandeep021**
issue #9170 fixed the visibilty of slide in dark theme
Steps to check:
1. Select "Settings" of the application
2. Select "Contacts and Calendar backup"

### CODE REVIEW

File: src/main/res/layout/backup_fragment.xml [hunk 1]
```diff
@@ -44,6 +44,7 @@
             android:minHeight="48dp"
             android:text="@string/contacts"
             android:textColor="@color/text_color"
+            app:theme="@style/Color1SwitchStyle"
```
Comment (AlvaroBrey) [inline src/main/res/layout/backup_fragment.xml:47]: The changes in this file and the other XML seem to be unneeded. Is that right? If so, remove them.

**Review by AlvaroBrey — CHANGES_REQUESTED** [review]
Thanks for your contribution @sandeep021! There are some things that need attention, please check the comments below.

File: src/main/res/layout/backup_fragment.xml (hunk 1 above)
Comment (sandeep021) [reply src/main/res/layout/backup_fragment.xml]: I have made all the suggested changes Thank you for the review and suggestions

File: src/main/java/com/owncloud/android/ui/fragment/contactsbackup/BackupFragment.java [hunk 2]
```diff
@@ -138,7 +138,7 @@ public View onCreateView(@NonNull final LayoutInflater inflater, ViewGroup conta
         arbitraryDataProvider = new ArbitraryDataProvider(getContext().getContentResolver());
 
         ThemeCheckableUtils.tintSwitch(
-            binding.contacts, ThemeColorUtils.primaryAccentColor(getContext()));
+            binding.contacts, getContext().getResources().getColor(R.color.color_accent));
```
Comment (AlvaroBrey) [inline src/main/java/com/owncloud/android/ui/fragment/contactsbackup/BackupFragment.java:141]: Is this change needed?

Comment (sandeep021) [reply src/main/java/com/owncloud/android/ui/fragment/contactsbackup/BackupFragment.java:141]: actually no both are doing same thing

File: src/main/java/com/owncloud/android/utils/theme/ThemeCheckableUtils.java [hunk 3]
```diff
@@ -55,16 +57,23 @@ public static void tintCheckbox(int color, AppCompatCheckBox... checkBoxes) {
     }
 
     public static void tintSwitch(SwitchCompat switchView, int color) {
```
Comment (AlvaroBrey) [inline src/main/java/com/owncloud/android/utils/theme/ThemeCheckableUtils.java:59]: the `color` param is now unused, so you can remove it to fix the Spotbugs increase.

Additionally please follow https://github.com/nextcloud/android/pull/9622/checks?check_run_id=4705568539 to sign off your commits, and then we can merge  this after both things are done.

Comment (sandeep021) [reply src/main/java/com/owncloud/android/utils/theme/ThemeCheckableUtils.java]: - removed the color param
-i am not sure i have followed the second suggestion or not i am unable to push using terminal getting 403, if not done properly please let me know i will try again

**Review by AlvaroBrey — APPROVED** [review]

**Comment by AlvaroBrey** [comment]
Thanks @sandeep021!

**Pull request merged**